css-grid-1/implicit-grids/grid-layout-auto-columns-rows.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
```

Comment:
About the name of the file, please use "grid-" as prefix (instead of "grid-layout").
Also add "-001" as we might eventually add more patches with the same name.
I'd use something like: "grid-definition/grid-support-grid-auto-columns-rows-001.xhtml"